conda/meta.yaml

```
@@ -0,0 +1,34 @@
+{% set name = "zppy" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/E3SM-Project/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ab6dee537f8252a9f244f01b725046dd64748d68
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
+  entry_points:
+    - zppy = zppy.__main__:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - zppy
+  commands:
+    - zppy --help
```

```suggestion
test:
  requires:
    - pip
  imports:
    - zppy
  commands:
    - zppy --help
    - pip check
```
`pip check` is a nice thing to add.